src/components/common/MediaCard.css

```diff
@@ -0,0 +1,3 @@
+.MediaCardDescriptionText p {
```

Comment:
This file is not needed. We are now moving away from using `.css` files, to embedding the styles inline. See MediaCard line 19 for an example.